util/MySqlMigrations/Migrations/DatabaseContextModelSnapshot.cs

```diff
@@ -854,7 +854,7 @@ protected override void BuildModel(ModelBuilder modelBuilder)
 
                     b.HasIndex("GroupId");
 
-                    b.ToTable("CollectionGroups");
+                    b.ToTable("CollectionGroups", (string)null);
```

Comment:
❓ I don't get these database context model snapshot changes when running migrations locally?

I don't think we need any EF migrations for this now.

Reply:
I will have to edit this file manually, since i removed the migration with `dotnet ef migrations remove --force`, instead of reverting the changes in commits and this is what i was left with 🤷

Reply:
Fixed in 70b31f2

Reply:
Fixed in 70b31f2